### PR TITLE
Add Bookalope icon

### LINKED ISSUE #7644: Add Bookalope icon
### Brand Name

Bookalope

### Website

http://bookalope.net/

### Popularity Metric

Huh 🤔 

No Similarweb rank? https://www.similarweb.com/search/?q=bookalope

### Official Resources for Icon and Color

- Logo: https://bookalope.net/img/bookalope-logo.svg
- Name: https://bookalope.net/img/bookalope-logo.png

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by jenstroeger**
Bookalope is a SaaS platform for AI-assisted manuscript analysis, accessible ebook conversion, and more.

bookalope

**Issue:** closes #7644 

**Similarweb rank:** n/a and https://github.com/bookalope/
  

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description

Color and logo have been in use since around 2016 when the project started.

### CODE REVIEW

**Review by dirien — CHANGES_REQUESTED** [review]
Hi @jenstroeger,

Thanks for your contribution. Could you check the errors in the lint run please?

Best start is to check the contribution guidlines.

https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md

**Comment by jenstroeger** [comment]
@dirien thank you for the poke. I tried to fix the problems but I’m unable to interpret the “Test package” failures. Also, after following the CONTRIBUTION guidelines lint still fails and I’m not sure how to address these two failures.

I’d be grateful for suggestions!

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1538,6 +1538,12 @@
             "hex": "1D439C",
             "source": "https://commons.wikimedia.org/wiki/File:Boeing_full_logo.svg"
         },
+        {
+            "title": "Bookalope",
+            "slug": "bookalope",
```
Comment (dirien) [inline _data/simple-icons.json:1543]: Please remove the slug

**Review by dirien — CHANGES_REQUESTED** [review]
Can you check the LF/CR? And center the icon please (in inkscape I use 
image
)

Last thing, remove the `slug` field from the `simple-icons.json`

**Comment by jenstroeger** [comment]
> Can you check the LF/CR?

It wasn’t that, but `hexdump -C bookalope.svg` showed a trailing LF—removed.

> And center the icon please (in inkscape I use image )

Done.

> Last thing, remove the `slug` field from the `simple-icons.json`

Done.

**Review by dirien — APPROVED** [review]
Perfect fit!

image

**Comment by jenstroeger** [comment]
@dirien oh great, thank you for merging! I look forward to the next release then 🤓

**Pull request merged**